workflow-manager/main.go

```diff
@@ -31,36 +31,44 @@ import (
-var gcpProjectID = flag.String("gcp-project-id", "", "Name of the GCP project ID being used for PubSub.")
-
-// Arguments for aws-sns task queue
-var awsSNSRegion = flag.String("aws-sns-region", "", "AWS region in which to publish to SNS topic")
-var awsSNSIdentity = flag.String("aws-sns-identity", "", "AWS IAM ARN of the role to be assumed to publish to SNS topics")
-
-// Define flags and arguments for other task queue implementations here.
-// Argument names should be prefixed with the corresponding value of
-// task-queue-kind to avoid conflicts.
+// Flags.
+var (
+	k8sNS                  = flag.String("k8s-namespace", "", "Kubernetes namespace")
+	ingestorLabel          = flag.String("ingestor-label", "", "Label of ingestion server")
+	isFirst                = flag.Bool("is-first", false, "Whether this set of servers is \"first\", aka PHA servers")
+	maxAge                 = flag.Duration("intake-max-age", time.Hour, "Max age (in Go duration format) for intake batches to be worth processing.")
+	ingestorInput          = flag.String("ingestor-input", "", "Bucket for input from ingestor (s3:// or gs://) (Required)")
+	ingestorIdentity       = flag.String("ingestor-identity", "", "Identity to use with ingestor bucket (Required for S3)")
+	ownValidationInput     = flag.String("own-validation-input", "", "Bucket for input of validation batches from self (s3:// or gs://) (required)")
+	ownValidationIdentity  = flag.String("own-validation-identity", "", "Identity to use with own validation bucket (Required for S3)")
+	peerValidationInput    = flag.String("peer-validation-input", "", "Bucket for input of validation batches from peer (s3:// or gs://) (required)")
+	peerValidationIdentity = flag.String("peer-validation-identity", "", "Identity to use with peer validation bucket (Required for S3)")
+	pushGateway            = flag.String("push-gateway", "", "Set this to the gateway to use with prometheus. If left empty, workflow-manager will not use prometheus.")
+	dryRun                 = flag.Bool("dry-run", false, "If set, no operations with side effects will be done.")
+	taskQueueKind          = flag.String("task-queue-kind", "", "Which task queue kind to use.")
+	intakeTasksTopic       = flag.String("intake-tasks-topic", "", "Name of the topic to which intake-batch tasks should be published")
+	aggregateTasksTopic    = flag.String("aggregate-tasks-topic", "", "Name of the topic to which aggregate tasks should be published")
+	maxEnqueueWorkers      = flag.Int("max-enqueue-workers", 100, "Max number of workers that can be used to enqueue jobs")
+
+	// Aggregation window flags, which determine which aggregation window will be aggregated (if not already aggregated).
+	// Normally, aggregation occurs for the window including the point in time of (now - grace-period).
+	// If aggregation-override-timestamp is specified, the aggregation window containing the override point will be aggregated instead.
```

Comment:
```suggestion
	// Aggregation window flags, which determine which aggregation window will be aggregated (if not already aggregated).
	// Normally, aggregation occurs for the window including the point in time of (now - grace-period).
	// If aggregation-override-timestamp is specified, the aggregation window containing the override point will be aggregated instead of the most recent aggregation window.
```

Reply:
Done.